app/views/startUrl.html

```diff
@@ -36,6 +38,26 @@
             </div>
             <a href="" class="label label-default" ng-click="start.autoIndent()">Indent JSON</a>
           </div>
+
+          <div class="tab-pane" id="os_">
+            <div class="input-group">
+              <input type="text" class="form-control" ng-model="start.osSettings.startUrl" ng-init="start.osSettings.startUrl='http://opensearch'">
```

Comment:
The start URL should be this demo OS server: https://github.com/o19s/quepid/wiki/Troubleshooting-Opensearch-and-Quepid#demo-opensearch-indexes-with-the-tmdb-dataset